content/posts/weekly/January-2023.md

```diff
@@ -13,3 +13,15 @@ weight: 57
 - Packit now puts the correct release number into the changelog when the
   Release tag is reset during propose-downstream.
   ([packit#1816](https://github.com/packit/packit/pull/1816)))
+
+## Week 1 (January 3rd – January 9th)
+
+- SRPMs for projects that installed the GitHub App before September 6th and
+  don't have `srpm_build_deps` defined are now built in Copr as well.
+  ([packit-service#1822](https://github.com/packit/packit-service/pull/1822))
+- We have fixed a bug in dashboard that linked `null` as a Copr build for
+  Testing Farm runs that do not require any Copr build.
+  ([dashboard#200](https://github.com/packit/dashboard/pull/200))
+- All classes including `Specfile` itself can now be copied using the standard
+  `copy()` and `deepcopy()` functions from `copy` module.
+  ([specfile#176](https://github.com/packit/specfile/pull/176))
```

Comment:
it's certainly not related to the Packit GitHub app but I think it's worthy to leave it here so everyone can see changes in our complete stack